tools/qm-storage-settings

```diff
@@ -38,18 +37,18 @@ exec_command() {
     fi
 
     # print the output of QM ROOTFS
-    echo $($COMMAND)
+    $COMMAND
 }
 
 setup_init() {
     ROOTFS=$(exec_command "/usr/share/qm/qm-rootfs")
 
     # containers/storage.conf
     STORAGE_CONFIG="${ROOTFS}/etc/containers/storage.conf"
-    if [ ! -f "${STORAGE_CONFIG}" ]; then # no path, let's try ostree approach
+    if [  -d /run/osbiuild/ostree ]; then # osbuild exist, let's try ostree approach
         STORAGE_CONFIG="/run/osbuild/tree/${STORAGE_CONFIG}"
         if [ ! -f "${STORAGE_CONFIG}" ]; then
-	    echo "Exiting /etc/containers/storage.conf in the ROOTFS"
+            echo "Exiting /etc/containers/storage.conf in the ROOTFS"
             exit 255
```

Comment:
Hi @Yarboa to my eyes, As we changed again the approach how to install (no more calling it via sample-images, no need to care about supporting os-tree / osbuild) we can just remove it and add a commit message explaning we removed so later if we need it back we can just "git log | grep <some words> and find it? "

Reply:
I tend to leave it for future use in case needed

Reply:
It will be dead code in the end. I think it's safe to remove, we will have git history to check later and AFAIK there is no plan to get setup as official.

Reply:
shall i remove this  from `qm.spec`
%{_datadir}/qm/qm-rootfs

Reply:
Not for now, I expect to have more tools around as much as we grow. Not meant, ways to installation :D